Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Beckn/OnSelect.hs

```diff
@@ -11,12 +11,15 @@
 
  the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
 -}
+{-# LANGUAGE TypeApplications #-}
 
 module Domain.Action.Beckn.OnSelect
   ( module Domain.Action.Beckn.OnSelect,
   )
 where
 
+import qualified Beckn.ACL.Init as ACL
+import qualified Domain.Action.UI.Confirm as DConfirm
```

Comment:
Move the function imported from here to SharedLogic and import it from there. We should not import any Domain/UI function inside Domain/Beckn.